src/database/migrations/0006_create_application_schema_fields.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE application_schema_fields (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  application_schema_id INTEGER,
+  canonical_field_id INTEGER,
```

Comment:
Do we want a unique constraint across `application_schema_id` and `canonical_field_id` as well?